src/cryptonote_core/tx_pool.h

```diff
@@ -62,24 +65,33 @@ namespace cryptonote
   //! pair of <transaction fee, transaction hash> for organization
   typedef std::pair<std::pair<double, std::time_t>, crypto::hash> tx_by_fee_and_receive_time_entry;
```

Comment:
Pretty sure that typedef is not needed anymore and can be deleted, in the interest of clean code?

Reply:
Thanks for your comment. Yes. it was not needed. But didn't want to make any unnecessary changes.